bluesky_database/backend/redis_testing/run_end_to_end_test.sh

```
@@ -0,0 +1,354 @@
+                    log_success "Service '$service_name' container is running (no HEALTHCHECK)"
+                    break
+                fi
+            fi
+        fi
+        
+        local now
+        now=$(date +%s)
+        local elapsed
+        elapsed=$(( now - start_time ))
+        if [ $elapsed -ge $timeout_seconds ]; then
+            log_error "Timed out waiting for '$container_name' to become healthy"
```

Shellcheck (0.10.0)

[warning] container_name is referenced but not assigned.

(SC2154)